packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -108,6 +113,10 @@ export abstract class TelemetryLogger implements ITelemetryLogger {
         if (event.stack === undefined && fetchStack) {
             event.stack = generateStack();
         }
+        // Tag the error stack trace as a CodeArtifact (see issue #6603)
+        if (event.stack) {
+            event.stack = { tag: TelemetryDataTag.CodeArtifact, value: event.stack};
```

Comment:
I think this change should be done when we remove PackageData. Basically, using new tag when app is not ready will result in app dropping this property, and we will not have that data for a while.